app/javascript/dashboard/routes/dashboard/settings/reports/components/ReportFilters.vue

```diff
@@ -185,6 +185,21 @@ export default {
       const fromDate = subDays(new Date(), diff);
       return this.fromCustomDate(fromDate);
     },
+    multiselectLabel() {
+      if (this.type === 'agent') {
+        return this.$t('AGENT_REPORTS.FILTER_DROPDOWN_LABEL');
+      }
+      if (this.type === 'label') {
+        return this.$t('LABEL_REPORTS.FILTER_DROPDOWN_LABEL');
+      }
+      if (this.type === 'inbox') {
+        return this.$t('INBOX_REPORTS.FILTER_DROPDOWN_LABEL');
+      }
+      if (this.type === 'team') {
+        return this.$t('TEAM_REPORTS.FILTER_DROPDOWN_LABEL');
+      }
+      return this.$t('INBOX_REPORTS.FILTER_DROPDOWN_LABEL');
```

Comment:
A switch case or an object is better in this case. For eg:

```js
multiselectLabel() {
  const typeLabels = {
    agent: this.$t('AGENT_REPORTS.FILTER_DROPDOWN_LABEL'),
    label: this.$t('LABEL_REPORTS.FILTER_DROPDOWN_LABEL');
    // ..
  };
  return typeLabels[this.type] || '';
}
```

Reply:
Resolved in https://github.com/chatwoot/chatwoot/pull/3116/commits/ac0c18a200d56fea549ababc00af74991f937120